src/encoded/schemas/patient.json

```diff
@@ -141,137 +126,129 @@
             "title": "Metastasis Status"
         },
-        "metastasis.histology_proven": {
-            "title": "Metastasis Histology Proven"
-        },
         "metastasis.site": {
             "title": "Metastatic Site"
         },
-        "metastasis.source": {
-            "title": "Metastasis Source"
-        },
-        "labs_and_vitals.BMI": {
-            "title": "BMI"
-        },
-        "labs_and_vitals.BP_Systolic": {
-            "title": "BP_Systolic (mmHg)"
+        "metastasis.histology_proven": {
+            "title": "Metastasis Histology Proven"
         },
-        "labs_and_vitals.BP_Diastolic": {
-            "title": "BP_Diastolic (mmHg)"
+        "surgery.pathology_report.metasis_details.site": {
+            "title": "Metastatic Site (Pathology)"
         },
-        "labs_and_vitals.Hemoglobin": {
-            "title": "Hemoglobin (g/dL)"
+        "vital_status": {
+            "title": "Vital Status"
         },
-        "labs_and_vitals.Platelets": {
-            "title": "Platelets (10^3/ul)"
+        "diagnosis.follow_up_duration_range": {
+            "title": "Follow Up Duration"
         },
-        "labs_and_vitals.WBC": {
-            "title": "WBC (10^3/ul)"
+        "medications.name":{
+            "title":"Medication"
         },
-        "labs_and_vitals.Neutrophils": {
-            "title": "Neutrophils (10^3/ul)"
+        "medication_range":{
+            "title":"Medication Duration"
         },
-        "labs_and_vitals.Creatinine": {
-            "title": "Creatinine (mg/dL)"
+        "supportive_medication.name":{
+            "title":"Supportive Medication"
         },
-        "labs_and_vitals.Calcium": {
-            "title": "Calcium (mg/dL)"
+        "radiation_summary": {
+            "title": "Radiation Treatment Status"
         },
-        "labs_and_vitals.Albumin": {
-            "title": "Albumin (g/dL)"
+        "radiation.site_consolidated" : {
+          "title": "Radiation Site"
         },
-        "labs_and_vitals.Sodium": {
-            "title": "Sodium (mmol/L)"
+        "radiation.dose_range": {
+            "title": "Radiation Dose per Fraction"
         },
-        "labs_and_vitals.LDH": {
-            "title": "LDH (U/L)"
+        "radiation.fractions_range": {
+            "title": "Radiation Fractions"
         },
-
         "biospecimen.sample_type": {
               "title": "Specimen Class"
         },
         "biospecimen.tissue_derivatives": {
             "title": "Specimen Type"
         },
-        "biospecimen.tissue_type": {
-            "title": "Specimen Pathological Type"
-        },
-        "biospecimen.anatomic_site_display": {
-            "title": "Specimen Anatomic Site"
-        },
         "biospecimen.species": {
             "title": "Specimen Species"
         },
-        "biospecimen.activity_status": {
-            "title": "Specimen Activity Status"
+        "biospecimen.biofile.assay_term_name": {
+            "title": "Genomics Assay"
```

Comment:
This one should be Genomics Availability. Do we have this facet? May need to add a calculated field for it.

Reply:
@mingjiecn  I think knowing the type of assay might be more helpful here than just a yes no. What do you think?

Reply:
You are the expert here. So totally agree!

Reply:
This field we should show genomics data related to the patient, I think it's fine to show what kind of genomic assay type we have in database.